File: contest-2/c++/p11-Palindrome between a Range.cpp
// Print palindromes between the range.

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
   int a,b,x=0,c;
    string y,z;
    cin>>a>>b;
    a++;
    while(a<b)
    {
        y=to_string(a);
        z=y;
        reverse(y.begin(),y.end());
        if(y==z)
        {
            x++;
            cout<<a<<" ";
        }
        a++;
    }
    if(x==0)
        cout<<"No Palindrome Numbers";
    return 0;
}

File: contest-2/python/p11-Palindrome between a Range.py
# Print palindromes between the range.

l=[]
l=list(input().split())
a=int(l[0])
b=int(l[1])
x=0
for i in range(a+1,b):
    s=str(i)
    s1=""
    l=len(s)
    for j in range(1,l+1):
        s1=s1+s[l-j]
    if(s==s1):
        x+=1
        print(i,end=" ")
if(x==0):
    print("No Palindrome Numbers")
